scripts/lisa-github-rulesets.sh

```
@@ -344,32 +344,95 @@ apply_ruleset() {
+  app_ids=$(echo "$payload" | jq --argjson actions "$ACTIONS_INTEGRATION_ID" \
+    '[.rules[]? | select(.type=="required_status_checks") | .parameters.required_status_checks[]?.integration_id | select(. != null and . != $actions)] | unique | .[]')
+
+  local drop_filter='
+    .rules |= (
+      map(
+        if .type == "required_status_checks" then
+          .parameters.required_status_checks |= map(select((.integration_id // -1) as $i | ($drop | index($i)) | not))
+        else . end
+      )
+      | map(select(.type != "required_status_checks" or (.parameters.required_status_checks | length) > 0))
+    )'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)